src/Spout/Writer/WriterAbstract.php

```diff
@@ -137,6 +154,16 @@ public function openToBrowser($outputFileName)
         $this->globalFunctionsHelper->header('Cache-Control: max-age=0');
         $this->globalFunctionsHelper->header('Pragma: public');
 
+        /*
+         * Set custom Headers
+         * Sometimes need to output or cover more headers.
+         *
+         * @see https://github.com/box/spout/issues/745
+         */
+        foreach ($headers as $header) {
```

Comment:
I would remove this from the PR. Adding custom headers should not necessarily be done this way.

Reply:
OK.The focus is `filename`.